.github/workflows/docs.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish documentation
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  publish-documentation:
+    permissions:
+      id-token: write
+      pages: write
+      contents: write
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set git user
+        uses: fregante/setup-git-user@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Generate Api docs
+        run: |
+          npm ci
+          npm run doc
+      - name: Push updated markdown docs
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: 'Update api markdown docs'
+          file_pattern: 'docs/api/md/*'
```

Comment:
Do we want to update the Markdown docs after every commit to `main`? I feel like...maybe not? I like that you've added it as part of the release task.

Reply:
Would you mind removing this, then?